### ISSUE TITLE
Enable other HRF models on front-end?

### ISSUE THREAD

**Issue by rbroc** [neuroscout/neuroscout#701]
Probably no big difference between glover and spm models, but FIR could be a useful add-on (cons: might require substantial changes to contrast specification)

**Comment by adelavega** [comment]
Yep, need to add F-tests that group the input HRF basis functions for the FIR.

This is going to take some thinking to get it right, and not clutter the interface.

**Comment by PeerHerholz** [comment]
Other models to consider: `sparse` and `multiband/fast`. As far as I can tell most naturalistic experiment use "classic" `TRs` (`1`-`3` seconds), there might be some that differ from that which might create problem for the "classic" models. While I would assume that `sparse` designs won't become a thing (however, there's a neat [nipype implementation](https://nipype.readthedocs.io/en/latest/api/generated/nipype.algorithms.modelgen.html#specifysparsemodel), ref [here](https://www.frontiersin.org/articles/10.3389/fnins.2013.00055/full). Protocols with `TR > 1 s` might appear more often in the future. While there are also some things that folks tried during preprocessing (e.g. different `bandpass` filters) which shouldn't be part of this discussion, the `temporal autocorrelation` is still a thing in the `GLM`. I think `SPM` has the `FAST` option for that, but I'm not sure how other packages handle/tackle these things (if at all). Obviously still low priority and introducing a bunch of changes, just wanted to add it.
